tests/saver/db/test_save_model_results_to_db.py

```diff
@@ -152,13 +151,17 @@ def test_write_temp_draws_file_and_upload_model_results_no_hdf_no_sr_call(
     draws_df, execution_context, fake_write_hdf
 ):
 
-    model_version_id = _write_temp_draws_file_and_upload_model_results(draws_df, execution_context, saver=save_results_fake)
+    model_version_id = _write_temp_draws_file_and_upload_model_results(draws_df,
```

Comment:
I'm less fond of hanging arguments near the edge. Have you considered either a) doing a return right after the left-paren and lumping them in there in one line or b) doing a return after the left-paren and hanging them there? This hanging stuff means that, when we change anything in the first line, we have to reformat the rest of the lines, and, while there's a keystroke for that, it seems a waste. It's also visually jarring when successive calls have hanging values at different columns, so you get a river-winding look.

Reply:
Sure